### PR TITLE
homework-0009

### PR DESCRIPTION
**Pull request by paulrenenichols**
- adding lesson 7 homework

### CODE REVIEW

File: apps/07jquery-tic-tac-toe/index.html [hunk 1]
```diff
@@ -23,7 +23,7 @@
   </div>
   <div id="announce-winner"></div>
   <button id="clear">Clear Board</button>
-  <script type="text/javascript" src="../vendor/jquery.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/2.1.4/jquery.js"></script>
```
Comment (tjefferson08) [inline apps/07jquery-tic-tac-toe/index.html:26]: Was this intentional? Makes sense to me to stick to the vendored version.

Comment (tjefferson08) [inline apps/07jquery-tic-tac-toe/index.html:26]: Or move it up a directory, now that I see it's been relocated

File: homework/07LessonSeven/index.html [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  <!-- attribute selectors -->
+  <div data-id="catbug">I'm Catbug!</div>
+  <div data-price="4"></div>
+
+  <!-- dom manipulation -->
+  <div id="insert-text"></div>
+  <div id="add-attribute"></div>
+  <div id="data-element" data-content="Text Content" data-id="data-example"></div>
+
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/2.1.4/jquery.js"></script>
```
Comment (tjefferson08) [inline homework/07LessonSeven/index.html:39]: Ditto here. Is there a reason to deviate from the vendored jquery?

File: apps/07jquery-tic-tac-toe/index.html (hunk 1 above)
Comment (paulrenenichols) [reply apps/07jquery-tic-tac-toe/index.html:26]: @tjefferson08 When i cd into the tic tac toe directory, and run `http-server`, the vendor version doesn't work.

I may have done something wrong. Would like you to comment on this.

**Pull request merged**